### PR TITLE
chore: Update v2 defaults in cloud run

### PR DESCRIPTION
**Pull request by qz267**

### CODE REVIEW

File: modules/v2/metadata.display.yaml [hunk 1]
```diff
@@ -115,6 +115,14 @@ spec:
         vpc_access:
           name: vpc_access
           title: Vpc Access
+          alt_defaults:
+            - type: ALTERNATE_TYPE_DC
+              value:
```
Comment (q2w) [inline modules/v2/metadata.display.yaml:120]: We need the value to be yaml equivalent of `{ network_interfaces : { network : "default", subnetwork : "default" } }`.   I think it should be
```
          alt_defaults:
            - type: ALTERNATE_TYPE_DC
              value:
                network_interfaces:
                  network: default
                  subnetwork: default
```

Comment (qz267) [reply modules/v2/metadata.display.yaml:120]: sgtm

**Review by q2w — APPROVED** [review]

**Comment by qz267** [comment]
```Step #3 - "create standalone": t=2024-10-18T10:07:19+0000 lvl=dbug msg="running /usr/bin/go with args [go test ./secure_cloud_run_standalone -run TestSecureCloudRunStandalone -p 1 -count 1 -timeout 0 -v] in /workspace/test/integration"
Step #3 - "create standalone": === RUN   TestSecureCloudRunStandalone
Step #3 - "create standalone": TestSecureCloudRunStandalone 2024-10-18T10:07:21Z command.go:100: Running command gcloud with args [access-context-manager policies list --organization=943740911108 --format value(name)]
Step #3 - "create standalone":     gcloud.go:110: Error parsing output, invalid json: 
Step #3 - "create standalone": --- FAIL: TestSecureCloudRunStandalone (1.20s)
Step #3 - "create standalone": FAIL
Step #3 - "create standalone": FAIL	github.com/terraform-google-modules/terraform-example-foundation/test/integration/secure_cloud_run_standalone	1.257s
Step #3 - "create standalone": FAIL
Step #3 - "create standalone": t=2024-10-18T10:07:22+0000 lvl=eror msg="error running command: exit status 1"
Finished Step #3 - "create standalone"
```
hmm, invalid Json indicates the yaml fmt issue to me.

**Pull request merged**